Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -179,6 +185,7 @@ public struct Configuration: Equatable {
             }
             self.init(dict: dict)!
             configurationPath = fullPath
+            hash = yamlContents.hash
```

Comment:
this would also allow `Configuration` to conform to `Hashable` and for caching to work with configurations created without a YAML source.